docs/managing-policy.md

```diff
@@ -42,3 +42,8 @@ To receive notifications via third-party services, like Slack or email:
 
 When Semgrep CI next runs and finds a result, the configured services will receive the finding.
 
+# Fail Open/Close
```

Comment:
```suggestion
# Fail open/close
```

Also: I’m not sure about the terminology here but I would have thought to use "Fail closed" instead of Fail close"?